### DISCUSSION TITLE
Windows msi installer, All users, launch at startup and do not display welcome screen by default for all users

### DISCUSSION THREAD

**Discussion by staman81** [flameshot-org/flameshot#3347] [Q&A]
Hello,

I would love to use this nice software in our Windows 10 environment, but I can't figure out, how to set these two settings for all users by default 

- Launch at startup ( Flameshot will start with Windows startup / for every single user thats log on)
- do not display welcome screen (No welcome screen will be displayed after user logs on/apply for all users)

I can set this manually for logged on user, but this won't apply for any other users :(

Please, how to achieve this ? 

I don't see any documentation for msi installer (no parameters or transformation files where I can set this),  no way to configure this via some registry keys or some settings.ini / cfg , simply nothing that allow me customization of Flameshot for All users 

Please be advised :) 
Thank you
Stan

PS: Yes, I can achieve Launch at startup via registry key but If I want to configure any other FlameShot parameters for ALL Users, i'm lost..

**Comment by mmahmoudian** [comment]
Sorry, this was lost in the Github notifications, but you also send this as an email to which fortunately I already replied 😃 

I just paste the email I sent here just in case someone else have the same question:

> Unfortunately none of the devs are Windows users, therefore we cannot
> answer you directly. However, Flameshot uses Qt framework and the
> settings should be at one of the locations indicated under
> AppConfigLocation key in this page:
> https://doc.qt.io/qt-5/qstandardpaths.html#StandardLocation-enum Also
> take a look at https://github.com/flameshot-org/flameshot/issues/2422
> 
> As far as I can remember from when I last used Windows, your best bet
> is to set Flameshot as startup for all users using ActiveDirectory (if
> you are doing this on an enterprise setup), and copy the Flameshot
> config you want to all users' profiles.
> 
> Perhaps you can ask this Window-specific question in some Windows
> forum or Stackoverflow and see what others can suggest.
> 
> I'm sorry that we cannot be much of a help.


Perhaps in future we can be more helpful when some Windows users also join our developer team.
